README.md

```diff
@@ -1,4 +1,44 @@
 # bootstrap_go
 Lightweight bootstrap nodes for Golem network. Basic implementation in Go.
 
+# installation
+
+## prerequisites 
+- install `go` tools https://golang.org/doc/install  
+- create go workspace `mkdir ~/go`
+
+## get package and deps
+
+```
+go get github.com/golemfactory/bootstrap_go
+cd ~/go/src/github.com/golemfactory/bootstrap_go/
```

Comment:
`cd $GOPATH/src/github.com/golemfactory/bootstrap_go` is more accurate :)

Reply:
I assume no custom GOPATH at all